scripts/globalTypes.d.lua

```diff
@@ -5909,6 +5909,7 @@ declare class HandlesBase extends PartAdornment
 end
 
 declare class ArcHandles extends HandlesBase
+	Adornee: BasePart?
```

Comment:
I think this change needs to be made in a parent class. Adornee is inherited from PartAdornment